src/payment/PaymentPage.jsx

```diff
@@ -73,6 +77,7 @@ class PaymentPage extends React.Component {
         {loaded ? (
           <div className="row">
             <div className="col-6">
+              <CouponForm />
```

Comment:
Doesn't the coupon form show up inside the BasketSummary?

Reply:
I haven't yet put it in the right place, admittedly.  It's on my coupon todo list. :)